components/ws-manager/pkg/manager/monitor.go

```diff
@@ -791,8 +791,9 @@ func (m *Monitor) initializeWorkspaceContent(ctx context.Context, pod *corev1.Po
 	} else {
 		err = handleGRPCError(ctx, err)
 	}
-	wsType := pod.Labels[wsk8s.TypeLabel]
-	hist, errHist := m.manager.metrics.initializeTimeHistVec.GetMetricWithLabelValues(wsType)
+	wsType := strings.ToUpper(pod.Labels[wsk8s.TypeLabel])
```

Comment:
Why convert to upper case?

Reply:
to make it consistent with the other metrics: https://github.com/gitpod-io/gitpod/pull/10376#issuecomment-1141888616

It's easier to make that one uppercase considering that the rest is, too

Reply:
I see, I was confused by the screenshot in this PR description, probably missed to update it.